tests/test_array_type.py

```diff
@@ -0,0 +1,255 @@
+from typing import Iterable
+
+import numpy as np
+import pytest
+
+import pixeltable as pxt
+import pixeltable.exceptions as excs
+from pixeltable import type_system as ts
+from pixeltable.type_system import ArrayType, ColumnType, IntType
+
+from .utils import reload_catalog, validate_update_status
+
+
+class TestArrayType:
+    @pytest.mark.parametrize('do_reload_catalog', [False, True], ids=['no_reload_catalog', 'reload_catalog'])
+    def test_array_dtypes(self, do_reload_catalog: bool, init_env: None, reset_db: None) -> None:
```

Comment:
Just put this in `test_types.py`? Do we really need to introduce a new `.py` test file just for this?